Makefile

```diff
@@ -104,13 +126,18 @@ build-runner-image:
 	docker build -t $(RUNNER_IMAGE_NAME) $(DOCKER_BUILD_EXTRA_ARGS) .
 
 run-command-%:
-	docker run --rm -t --name $(DOCKER_DEV_CONTAINER_NAME) --network host $(DOCKER_RUN_MOUNT_OPTIONS) $(DOCKER_ENV) $(RUNNER_IMAGE_NAME) make $* EXTRA_ARGS=$(EXTRA_ARGS)
+	docker run --rm -t --name $(DOCKER_DEV_CONTAINER_NAME) --network host \
+	           $(DOCKER_RUN_MOUNT_OPTIONS) $(DOCKER_ENV) $(RUNNER_IMAGE_NAME) \
+	           make $* EXTRA_ARGS=$(EXTRA_ARGS)
 
 # Dev commands
 start-dev: build-runner-image localstack-start run-command-install-node-packages run-dev
 
-run-dev: run-command-build-node-all run-command-package-all run-command-tf-init run-command-tf-plan-apply run-command-tf-apply
+run-dev: run-command-build-node-all run-command-package-all run-command-tf-init \
+         run-command-tf-plan-apply run-command-tf-apply
+
```

Comment:
please remove extra line. One line gap, should be enough

Reply:
updated in b6b119a6dd73086ee41929ddbaedf2483cca9b9c